docs/tutorials/oss-to-enterprise-migration-new.md

```diff
@@ -0,0 +1,99 @@
+# Upgrading from Open Source to Enteprise Using a New Server
+
+Congratulations, you have decided to use Rundeck Enterpise! You are on you way to automating your workflows. Before that happens, you need to get your Rundeck Enterprise set up. Fear not, this guide will get you set up no matter what kind of Rundeck user you are. 
+
+## RPM Upgrade
+
+* Delete job execution history. If you do not, then the exporting of projects will take a much longer time.
```

Comment:
"Delete unnecessary job execution history.  By trimming this history exporting the jobs will proceed much faster and have you up and running sooner."

Language needs to be suggestion here and provide the positive reason why versus the negative.

Reply:
Updated